biggraphite/drivers/elasticsearch.py

```diff
@@ -567,6 +577,9 @@ def glob_directory_names(self, glob):
         # This may not be the same behavior as other drivers.
         # It returns the glob with the list of possible last component for a directory.
         # It doesn't return the list of fully defined directory names.
+        if "distinct_dirs" not in response.aggregations:
+            return []
```

Comment:
When does it happen?

If that is not an expected behavior, what do you think about logging a warning?

Reply:
This happend when there is no index to query following the search. Therefore it's a normal and expected behavior (empty cluster => empty result). I would not add a warning, but I added a comment.